### PR TITLE
Integrate gutenberg-mobile release 1.47.0

### PR DESCRIPTION
**Pull request by antonis**
## Description
This PR incorporates the 1.47.0 release of gutenberg-mobile.  
For more information about this release and testing instructions, please see the related Gutenberg-Mobile PR: https://github.com/wordpress-mobile/gutenberg-mobile/pull/3155
Release Submission Checklist
- [x] I have considered if this change warrants user-facing release notes and have added them to \`RELEASE-NOTES.txt\` if necessary.

### CODE REVIEW

**Review by dcalhoun — COMMENTED** [review]
This is looking good to me so far. Hopefully the testing build finishes soon.

Based on https://github.com/wordpress-mobile/WordPress-iOS/pull/15752#pullrequestreview-584197256, we'll need to swap from a git commit to a tag after we merge wordpress-mobile/gutenberg-mobile#3155.

File: Podfile [hunk 1]
```diff
@@ -153,7 +153,7 @@ target 'WordPress' do
     ## Gutenberg (React Native)
     ## =====================
     ##
-    gutenberg :tag => 'v1.47.0-alpha2'
+    gutenberg :commit => '7f75b334f402ff36ac154274b728c25533dcb3a4'
```
Comment (dcalhoun) [inline Podfile]: Prior to merging, we'll need to create a `v1.47.0` in `gutenberg-mobile`, reference that here, and install the pods.

**Review by dcalhoun — APPROVED** [review]
LGTM. We just need to change the Podfile ref from commit to tag before merging.

File: RELEASE-NOTES.txt [hunk 2]
```diff
@@ -2,6 +2,10 @@
 -----
 * [**] Prevent deleting published homepages which would have the effect of breaking a site. [#15797]
 * [**] Prevent converting published homepage to a draft in the page list and settings which would have the effect of breaking a site. [#15797]
+* [*] Block Editor: Fixed block mover title wording for better clarity from 'Move block position' to 'Change block position'. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3049]
+* [**] Block Editor: Make inserter long-press options "add to beginning" and "add to end" always available. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3074]
```
Comment (fluiddot) [inline RELEASE-NOTES.txt]: 👋 This change was included in the `1.46.1` beta fix so it should better go under `16.7` section.

File: RELEASE-NOTES.txt [hunk 3]
```diff
@@ -2,6 +2,10 @@
...
+* [**] Block Editor: Add support for setting Cover block focal point. [https://github.com/wordpress-mobile/gutenberg-mobile/pull/3028]
+* [*] Block Editor: Fix crash when Column block width attribute was empty. [https://github.com/WordPress/gutenberg/pull/29015]
```
Comment (fluiddot) [inline RELEASE-NOTES.txt]: The same here, this was included in `1.46.1` beta fix so better to have it under `16.7` section.

**Review by fluiddot — COMMENTED** [review]
👋 I've found some discrepancies in the release notes related to the changes included the beta fixes.

**Review by fluiddot — APPROVED** [review]

**Pull request merged**